sap/latest/support/index.md

```diff
@@ -6,8 +6,4 @@ The following are the supported platforms for Alfresco Content Connector for SAP
 
 | Version | Notes |
 | ------- | ----- |
-| Content Services 7.3.x | |
```

Comment:
Only remove 6.2.x, since the version is still backward compatible with previous 7.x versions

Reply:
@psdfrisch  I've just reverted the deletions, but now I'm wondering if I should change `7.4.1` to `7.4.x`?

Reply:
You are right, since there is already a 7.4.0. And future 7.4. minor releases should not break compatibility.